src/main/resources/com/google/api/codegen/go/example.snip

```diff
@@ -1,86 +1,87 @@
 @extends "go/header.snip"
 
-@snippet generateFilename(service)
-    {@context.getApiConfig.getPackageName}/{@context.getReducedServiceName(service)}_client_example_test.go
-@end
-
-@snippet generateClass(service, body)
+@snippet generate(view)
     {@headerComment()}
 
-    // AUTO-GENERATED CODE. DO NOT EDIT.
-
-    package {@context.getPackageName}_test
+    package {@view.localPackageName}_test
```

Comment:
MVVM is against string concatenation is this from. In this case though, I think there are good reasons for it.
- It lets us use essentially the same view for library and sample, the only differences being file names and imports.
- The concatenation is rigid. The godoc tool strictly requires that the example of `Type.Method` be called `ExampleType_Method`. Even though it is strictly speaking one token, it might as well be treated as three (`{Keyword "Example"} {Identifier "Type"} {Identifier "Method"}` separated by an empty string and an underscore) by the doc generation tool.

@garrettjonesgoogle Are you OK with this?